pkg/controllers/provisioningv2/rke2/machineprovision/controller.go

```diff
@@ -252,7 +283,7 @@ func getMachineStatusFromPod(pod *corev1.Pod, kind, condType string) rkev1.RKEMa
 	for _, containerStatus := range pod.Status.ContainerStatuses {
 		if containerStatus.State.Terminated != nil && containerStatus.State.Terminated.ExitCode != 0 {
 			failureMessage := strings.TrimSpace(containerStatus.State.Terminated.Message)
-			message := CreatingMachineMessage(kind)
+			message := failureMessage
```

Comment:
It was changed because the `CreatingMachineMessage` is for actively creating machines i.e. it says "creating machine"

Since the create failed at this point, the message should be changed. I updated the message as you suggested.